WordPress/wordpress-playground
# Browser: Rewrite target="_top" as target="wordpress_playground"

🚧  This PR needs a rebase  🚧 

WordPress is rendered in a sandboxed iframe that prevents top-level navigation.
However, some links in wp-admin, such as the "activate plugin" link
rendered after the plugin is uploaded, are rendered with
target="_parent".

It does not make sense to target the top frame in this context. What we really want
is to target the WordPress iframe.

This commit fixes that via a <script> tag that rewrites the `target` of all
clicked A elements from _top and _parent to wordpress-playground.

## Testing plan

1. Upload a plugin in wp-admin
2. Click "Activate plugin"
3. Confirm it worked and there are no errors in devtools
4. Play with other wp-admin pages and confirm no previously working links are now broken

Related: https://github.com/WordPress/wordpress-playground/issues/266

cc @ajitbohra @georgestephanis
